python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -174,7 +177,9 @@ def load_model():
         }
 
         model_config = AutoConfig.from_pretrained(
-            str(model_id_or_path), revision=kwargs.get("model_revision", None)
+            str(model_id_or_path),
+            revision=kwargs.get("model_revision", None),
+            trust_remote_code=kwargs.get("trust_remote_code", False),
```

Comment:
Is this parameter set to False if omitted?
Is there any risk of requiring extra configuration or breaking the initialization of other models?

Reply:
in general, `trust_remote_code` defaults to `False` in the AutoConfig class https://huggingface.co/docs/transformers/v4.37.2/en/model_doc/auto#transformers.AutoConfig.from_pretrained, so omitting this argument would just pass the existing default value in